distribution/lib/Standard/Base/0.0.0-dev/src/System/File.enso

```diff
@@ -497,8 +529,13 @@ type File
              import Standard.Examples
 
              example_is_directory = Examples.csv.is_directory
+
+       ? Data Links
+         If the file is a data link, this checks whether the target of the data 
+         link is a directory.
     is_directory : Boolean
-    is_directory self = @Builtin_Method "File.is_directory"
+    is_directory self =
+        Data_Link_Helpers.is_directory self self.is_directory_builtin
```

Comment:
Morning... finally I find the `File.is_directory`. I am glad to see the _public API_ method is no longer calling a builtin directly. 

However I think the `self.is_directory_builtin` would also be visible in the _public API_. Can you turn it into non-virtual method somewhere in a `private` module can call it like `Enso_File_Helpers.is_directory_builtin self`?